core/server/api_container/server/startosis_engine/kurtosis_starlark_framework/enclave_plan_capabilities/enclave_plan_capabilites_builder.go

```diff
@@ -0,0 +1,59 @@
+package enclave_plan_capabilities
+
+import (
+	"github.com/kurtosis-tech/kurtosis/container-engine-lib/lib/backend_interface/objects/service"
+)
+
+type EnclavePlanCapabilitiesBuilder struct {
+	instructionTypeStr string
+	serviceName        service.ServiceName
+	serviceNames       []service.ServiceName
```

Comment:
I don't understand those 2 fields... Can't we have only one `serviceNames` array, and eventually have a single element in it?